### ISSUE TITLE
Ruby driver no longer installs on Ruby 1.9

### ISSUE THREAD

**Issue by danielmewes** [rethinkdb/rethinkdb#6090]
The `json` gem that we depend on changed their minimum required version:
```
ERROR:  Error installing rethinkdb:
        json requires Ruby version ~> 2.0.
```

@larkost noticed that we might not need that dependency at all, since the `json` module has been made a standard component of Ruby a while back. We're currently testing this.

**Comment by larkost** [comment]
@danielmewes's and my testing concluded that we do not need this dependency to be explicitly marked, since this is now part of the stdlib in Ruby. Ignoring the dependency and removing it from runtime has tested as working, so we are removing this line.

This is in `next` with 17ec8876 and `v2.3.x` with 0df55924.

**Comment by danielmewes** [comment]
Released as Ruby driver version 2.3.0.2 https://rubygems.org/gems/rethinkdb/versions/2.3.0.2
